Modules/posixmodule.c

```diff
@@ -8126,7 +8126,8 @@ os_fork1_impl(PyObject *module)
         /* parent: release the import lock. */
         PyOS_AfterFork_Parent();
         // After PyOS_AfterFork_Parent() starts the world to avoid deadlock.
-        warn_about_fork_with_threads("fork1");
+        if (warn_about_fork_with_threads("fork1") < 0)
+            return NULL;
```

Comment:
Please use braces around `if` bodies:
```suggestion
        if (warn_about_fork_with_threads("fork1") < 0) {
            return NULL;
        }
```
(here and below)